packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -46,31 +65,52 @@ export default (props) => {
   delete customProps.className;
 
   return (
-    <section className={mainClasses} {...customProps}>
-      {props.photo && (
+    <section
+      {...customProps}
+      className={mainClasses}
+      tabIndex={-1}
+    >
+      {photo && (
         <div className={cx('profile-photo')}>
-          {props.photo}
+          {photo}
         </div>
       )}
       <div className={cx('content')}>
-        <div className={cx('row')}>
+        <div className={cx('column')}>
           <h1 className={cx('person-name')}>
+            { isConfidential ? <span className={cx('confidential-icon')}><IconFlag /></span> : null }
             { personName }
             { preferredFirstName && <span className={cx('preferred-first-name')}>{ preferredFirstName }</span> }
           </h1>
-          <div className={cx('application-content')}>
-            {applicationContent}
-          </div>
-        </div>
-        <div className={cx('row')}>
           <div className={cx('person-details')}>
             {DemographicsBannerUtils.personDetails(props)}
           </div>
+        </div>
+        <div className={cx('column')}>
+          <div className={cx('application-content')}>
+            {applicationContent}
+          </div>
           <div className={cx('identifiers')}>
             {DemographicsBannerUtils.applicationIdentifiers(props)}
           </div>
         </div>
       </div>
+      {isSelectable && (
+        <div
+          data-focus-styles-enabled
+          role="button"
+          aria-label={personName || intl.formatMessage({ id: 'Terra.demographicsBanner.info' })}
+          className={cx('info-tile-container')}
+          tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Does this eslint rule still need to be disabled with `role="button"`?

Reply:
It is no longer needed: https://github.com/cerner/terra-core/pull/2856/commits/5b98d5ff7de9b4397bea7b94cb00934e5930d1e5